WooCommerce/Classes/ViewRelated/ReusableViews/LeftImageTableViewCell.swift

```diff
@@ -52,4 +52,16 @@ extension LeftImageTableViewCell {
         imageView?.image = image
         textLabel?.text = text
     }
+
+    // Custom configure, including image size and text color
+    func configure(image: UIImage, imageSize: CGSize, text: String, textColor: UIColor) {
+        UIGraphicsBeginImageContext(imageSize)
+        image.draw(in: CGRect(x: 0, y: 0, width: imageSize.width, height: imageSize.height))
```

Comment:
❓ This works but seems to make the Blaze icon a bit unsharp from what I can see. Is there a way to improve this?

Reply:
Modern iPhones display images at scale 2x or 3x (depending on the models) - so when preparing the context for image redraw you need to take that into account:

```Swift
        UIGraphicsBeginImageContextWithOptions(imageSize, false, UIScreen.main.scale)
```

However, redrawing images creates some memory overhead, especially when you do it when configuring cells. I have 2 suggestions:
- Keep your redrawing method but put it in an extension of `UIImage`, and prepare the image as a constant in the data source class. That way, you only need to redraw it once. 
- You can use the new method [`preparingThumbnail`](https://developer.apple.com/documentation/uikit/uiimage/3750835-preparingthumbnail) when configuring the cell, but the resulting image is a bit blurry.

Reply:
Great suggestions, I followed the first one since the second requires iOS 15 and I'm not sure yet how to do it.

Handled this in https://github.com/woocommerce/woocommerce-ios/pull/10970/commits/6241a8b03da5d82d5d63caa11d877ed4b0d6d475